helm/prometheus-rules/templates/alerting-rules/prometheus-operator.rules.yml

```diff
@@ -36,7 +36,7 @@ spec:
       labels:
         area: empowerment
         cancel_if_outside_working_hours: "true"
-        severity: notify
+        severity: page
```

Comment:
I'm not sure regarding those Prometheus-operator alerts. 
they seem just normal warnings to me, 

Having a notification is just enough for this kind of alerts.
 maybe only email notification is enough. 

So, we can lower the priority of those alerts to P4, P5 ?

What do you think?

Reply:
For Atlas, there is no such thing as "a warning" "an email" or "a low priority alert".
Either it pages, or not, as far as I understood.

So, IMO
* if we keep it this way we can only see it in the prometheus list of alerts, and in alertmanager.
* or we make it page during WH, so we are informed when it goes wrong
* or we remove the check?

Reply:
Let’s stick with the current rules,
In future we can enhance it in Opsgenie.